tests/integration_tests/functional/test_net.py

```diff
@@ -50,3 +55,65 @@ def test_high_ingress_traffic(test_microvm_with_ssh, network_config):
     # ssh commands.
     exit_code, _, _ = ssh_connection.execute_command('echo success\n')
     assert exit_code == 0
+
+
+@decorators.test_context('ssh_and_balloon', NO_OF_VMS)
+def test_macvtaps(test_multiple_microvms):
+    """Check the MacVTap functionality."""
+    microvms = test_multiple_microvms
+    # Creating a bridge for the macvtap interfaces.
+    bridge = net_tools.Bridge("bridge{}".format(randint(1, 10000)))
```

Comment:
Do we need to `randint()` here? If we're trying to make sure that we don't create multiple bridges with the same name we could just check if the bridge exists and tear it down if it does.